New sort order of /blockstack page

hstove opened the pull request:
Based on #474 , this PR changes the order of the /blockstack page. It now pulls app mining data, and displays Blockstack apps based on that data. It also removes the 'tweets / week' column, since that is no longer relevant (only for this page).

aulneau commented on stores/apps/index.js:
```diff
@@ -121,14 +121,20 @@ const actions = {
-    return !!tags.find((tag) => tag.toLowerCase() === platform.toLowerCase())
-  })
+export const selectAppsForPlatform = (apps, platform, blockstackRankedApps) => {
+  // console.log(blockstackRankedApps)
```
We should just remove this.

aulneau commented on stores/apps/index.js:
```diff
@@ -155,14 +161,16 @@ const makeReducer = (data) => {
     switch (action.type) {
       case constants.SELECT_PLATFORM: {
-        console.log(action.type)
+        // console.log(action.type)
```
also this

hstove commented:
Thanks! Comments removed.

aulneau commented:
@hstove looks good!

markmhx commented:
Staging looks good to me, too 👍🏻

markmhx commented:
I'd suggest we also remove the "Blockchain" column given how we want this page to focus on Blockstack adoption specifically, and having that column just prompts the confusing question of "Oh, are these not using the _Stacks_ blockchain?".

aulneau commented:
> though I only see 7 apps presumably since this is staging data? Is there any way to cross-check that the right apps are being pulled?

@markmhx yes that is because it is staging data. You can pull this branch locally, run yarn && yarn dev and navigate to localhost:3000/blockstack and see the correct apps.

hstove commented:
I updated the staging link to pull from production's data, and removed the `blockchain` column.

https://app-co-staging-pr-502.herokuapp.com/blockstack

markmhx commented:
Looks great!

hstove commented:
This has been merged to production.
